services/oc-opencor-base/docker/custom/Dockerfile

```diff
@@ -0,0 +1,105 @@
+FROM opencor/opencor:2020-06-16 as base
```

Comment:
Just a quick headsup that I am going to release a new snapshot for OpenCOR sometime today, which will result in a new docker. So, you will need to update this to `opencor/opencor:2020-06-17`.

Reply:
Same in `services/oc-opencor-base/.cookiecutterrc`, I imagine...?

Reply:
will wait for the newer version then =)